cache.go

```diff
@@ -161,39 +174,65 @@ func (c *Cache) Delete(key string) bool {
 	return false
 }
 
-// Clears the cache
+// Clears the cache.
+// This is a control command.
```

Comment:
Might be good, in this and similar doc comments, to say something like "See Configuration.ControlBuffer" because that seems to be where "control command" is defined.